src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -1,60 +1,26 @@
 import React from 'react';
-import { node, string, bool, objectOf } from 'prop-types';
+import { node } from 'prop-types';
 import HeaderContainer from '../containers/Header';
 import FooterContainer from '../containers/Footer';
 import ManifestContainer from '../containers/Manifest';
 import ServiceWorkerContainer from '../containers/ServiceWorker';
-import { DialContextProvider } from '../contexts/DialContext';
-import { ServiceContextProvider } from '../contexts/ServiceContext';
-import { RequestContextProvider } from '../contexts/RequestContext';
 import ConsentBanner from '../containers/ConsentBanner';
 import GlobalStyle from '../lib/globalStyles';
 
-const PageWrapper = ({
-  bbcOrigin,
-  children,
-  id,
-  service,
-  isAmp,
-  pageType,
-  dials,
-}) => (
-  <DialContextProvider dials={dials}>
-    <ServiceContextProvider service={service}>
-      <GlobalStyle />
-      <RequestContextProvider
-        bbcOrigin={bbcOrigin}
-        id={id}
-        isAmp={isAmp}
-        pageType={pageType}
-        service={service}
-      >
-        <ServiceWorkerContainer />
-        <ManifestContainer />
-        <ConsentBanner />
-        <HeaderContainer />
-        {children}
-        <FooterContainer />
-      </RequestContextProvider>
-    </ServiceContextProvider>
-  </DialContextProvider>
+const PageWrapper = ({ children }) => (
+  <>
```

Comment:
Hey, i dont think we should mix our usage of `<Fragment>` and `<>` in simorgh, can this be `<Fragment>` for now, and raise an issue to make all uses the same across simorgh, if you want us to change to it please?

Reply:
Updated.